aspnetcore/fundamentals/dotnet-scaffold-telemetry.md

```diff
@@ -0,0 +1,70 @@
+---
+title: dotnet-scaffold telemetry
+author: tdykstra
+description: Learn about the telemetry collected by the dotnet-scaffold CLI tool.
+monikerRange: '>= aspnetcore-8.0'
+ms.author: tdykstra
+ms.date: 11/06/2024
+uid: fundamentals/dotnet-scaffold-telemetry
+---
+# dotnet-scaffold telemetry
+
+The dotnet-scaffold command includes a telemetry feature that collects usage data. This feature helps the dotnet-scaffold team understand how the tool is used so it can be improved.
+
+## How to opt out
+
+The dotnet-scaffold telemetry feature is enabled by default. To opt out of the telemetry feature, set the `DOTNET_SCAFFOLD_TELEMETRY_OPTOUT` environment variable to `1` or `true`.
+
+## Disclosure
+
+The dotnet-scaffold displays text similar to the following when you first run the tool. Text may vary slightly depending on the version of the tool you're running. This "first run" experience is how Microsoft notifies you about data collection.
```

Comment:
The first sentence here sounds a bit off, how about "When you first run the dotnet-scaffold tool, it displays output similar to the following."